packages/component/src/BasicTranscript.tsx

```diff
@@ -743,6 +755,7 @@ const InternalTranscriptScrollable: FC<InternalTranscriptScrollableProps> = ({
 
 InternalTranscriptScrollable.propTypes = {
   children: PropTypes.any.isRequired,
+  isWaiting: PropTypes.bool.isRequired,
```

Comment:
Maybe rename it to `aria-busy`.

```suggestion
  'aria-busy': PropTypes.bool.isRequired,
```

Reply:
Done.